sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/jit/ToJitInnerVisitor.java

```diff
@@ -456,66 +461,75 @@ public VisitDecision preorder(DBSPSomeExpression expression) {
         return VisitDecision.STOP;
     }
 
+    static class FT {
```

Comment:
This is better called `JitFunction` I think.

Reply:
Too much to type